docs/components/modeler/bpmn/user-tasks/user-tasks.md

```diff
@@ -55,6 +55,29 @@ The unique identifier depends on the authentication method used to login to Task
 For example, say you log into Tasklist using Camunda Platform 8 login with email using your email address `foo@bar.com`. Every time a user task activates with `assignee` set to value `foo@bar.com`, Tasklist automatically assigns it to you. You'll be able to find your new task under the task dropdown option `Claimed by me`.
 :::
 
+## Scheduling
+
+User tasks support specifying a task schedule using the `zeebe:taskSchedule` extension element.
+This can be used to define **when** users interact with a given task. One or both of the following
+attributes can be specified simultaneously:
+
+- `dueDate`: Specifies the due date of the user task.
+- `followUpDate`: Specifies the follow-up date of the user task.
+
+You can define the due date and follow-up date as static values (e.g. `2023-02-28T13:13:10+02:00`), but you can also use
+[expressions](/components/concepts/expressions.md) (e.g. `= schedule.dueDate` and `= now() + duration("PT15S")`). The
+expressions are evaluated on activating the user task and must result in a `string` conforming to an ISO 8601 combined
+date and time representation.
+
+:::note
```

Comment:
:wrench: What about adding an example use case as we did in assignments section? Would be quite helpful to clearly understand the new feature.

Reply:
That's a nice idea. Thanks @berkaycanbc.

Since this PR focuses only on the Zeebe part, I will add an example use-case on how users can use these dates with Zeebe, independently of Tasklist.